prompt2model/utils/api_tools.py

```diff
@@ -15,7 +15,7 @@
 from litellm import acompletion, completion
 from tqdm.asyncio import tqdm_asyncio
 
-OPENAI_ERRORS = (
+API_ERRORS = (
```

Comment:
I am pretty confused. If I use API to call Claude and this API call fails, will it return me an error with `openai.error`? Will it be `claude.error`?

Reply:
There's a mapping layer that maps all errors to openai exception types: https://github.com/BerriAI/litellm/commit/ef431415549dff307d1f312c32bdcbf43be3f3b4

Reply:
Sure! Should we add a comment here to explain this?